### PR TITLE
Cruise Ship Function

### PR DESCRIPTION
**Pull request by wooyunlong**
-Add Cruise Ships section to the home page, below Flights section.
-Cruise ship data April 16.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -291,8 +291,17 @@ function App() {
             <Header province={province} />
           </Grid>
           <Grid item xs={12} className="removePadding">
-            <Navbar setNav={setNav} nav={nav} />
-            {/*<Navbar  province={province} overall={overall} myData={myData} area={area} data={data} setProvince={setProvince} gspace={gspace} columns={columns}/>*/}
```
Comment (freddiewanah) [inline src/App.js:295]: should change back

File: src/Header.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
```
Comment (freddiewanah) [inline src/Header.js:1]: this is not necessary ?

File: src/Homepage.js [hunk 3]
```diff
@@ -0,0 +1,466 @@
...
+              onClick={name => {
+                const p = provincesByName[name];
+                if (p) {
+                  setProvince(p);
+                }
+              }}
+              newData={myData}
+            />
+            {/*{*/}
+            {/*province ? false :*/}
```
Comment (freddiewanah) [inline src/Homepage.js:61]: remove?

File: src/App.js (hunk 1 above)
Comment (wooyunlong) [reply src/App.js]: I have changed it back

File: src/Header.js (hunk 2 above)
Comment (wooyunlong) [reply src/Header.js]: My branch was a bit out of date, so it remained some old files that's already deleted from the latest dev branch. The merge from Dev didn't remove such files from my branch. I have removed them manually.

File: src/Homepage.js (hunk 3 above)
Comment (wooyunlong) [reply src/Homepage.js]: My branch was a bit out of date, so it remained some old files that's already deleted from the latest dev branch. The merge from Dev didn't remove such files from my branch. I have removed them manually.

**Comment by wooyunlong** [comment]
The branch was a bit out of date, so some old files (deleted from the latest dev branch) are still there. I have reworked the files on my branch based on the latest dev, so now only 3 files on the pr have been changed. A brand new Ships.js. A homepage.js, I deleted 3 imports that's not being used anymore. A .gitignore file that I accidentally committed when I opened pr, and I changed it back.

**Pull request merged**